### ISSUE TITLE
redundantFileprivate breaks visibility of synthesised struct init methods

### ISSUE THREAD

**Issue by kevinoneill** [nicklockwood/SwiftFormat#393]
before
```
struct A {
  fileprivate let string: String
}

let a = A(string: "test")
```

after 

```
struct A {
  private let string: String
}
```

The initialiser is no longer visible so
```
let a = A(string: "test")
```

wont compile.

**Comment by nicklockwood** [comment]
@kevinoneill thanks for reporting, I'll get that fixed asap.

**Comment by nicklockwood** [comment]
@kevinoneill fixed in 0.40.3.

**Comment by kevinoneill** [comment]
looks good now. thanks @nicklockwood
